dependencies.yaml

```
@@ -130,12 +136,16 @@ dependencies:
       - output_types: [conda, requirements]
         packages:
           - pre-commit
-  doc:
+  docs_build:
     common:
       - output_types: [conda, requirements]
         packages:
           - pydata-sphinx-theme
           - myst-parser
+          - doxygen
+          - numpydoc
+          - ipython
+          - nbsphinx
```

```suggestion
          - doxygen
          - ipython
          - myst-parser
          - nbsphinx
          - numpydoc
          - pydata-sphinx-theme
```

Let's alphabetize these to keep things neat.

FWIW, I don't alphabetize these by hand.

If you highlight the lines in VS Code, you can use the `Sort Lines Ascending/Descending` commands in the [command palette](https://docs.github.com/en/codespaces/codespaces-reference/using-the-vs-code-command-palette-in-codespaces#accessing-the-vs-code-command-palette).